src/Skia/Avalonia.Skia/FormattedTextImpl.cs

```diff
@@ -331,7 +334,7 @@ private static bool IsBreakChar(char c)
             {
                 float measuredWidth;
                 string subText = textInput.Substring(textIndex, stop - textIndex);
-                lengthBreak = (int)paint.BreakText(subText, maxWidth, out measuredWidth) / 2;
+                lengthBreak = (int)paint.BreakText(subText, maxWidth, out measuredWidth);
```

Comment:
Was this something we were doing incorrectly before, or did something in the semantics of `BreakText` change between Skia versions?